hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -227,6 +244,39 @@ def _collect_folder(self, parent_item, folder):
 
             file_items.append(file_item)
 
+        # recurse through any subfolders
```

Comment:
We discussed this internally yesterday. The consensus was that, while awesome and powerful, it isn't something we are ready to commit to in terms of support and general workflow for our shipped plugins. At least for now, we'd like to stick with the simpler case of processing only the first level of files in a dropped folder. 

That said, it is really, really great to see that you're doing this and that it is (hopefully) working well for you. This is exactly the type of customization we were hoping studios would implement when we wrote these basic hooks.